docs/hardware/01-light/case.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: "Case"
+description: "The Body of the 01 Light"
+---
+
+# Case
+
+You can 3D print the 01 case at home. For the best quality, it's recommended to use a resin printer.
+
+Watch this video from James at CAD9 Design for a deep dive on his design.
+
+<iframe
+  width="560"
+  height="315"
+  src="https://www.youtube.com/embed/BjoO0Kt-IWM"
+  frameBorder="0"
+  allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+  allowfullscreen
+></iframe>
+
+The STL files can be found [here](https://github.com/OpenInterpreter/01/tree/main/hardware/light/bodies)
```

Comment:
Period at the end (ik this can be stylistic, but I think official docs should always follow convention).